### PR TITLE
Added tests for f-test

### PR DESCRIPTION
**Pull request by ssurbhi560**
Added some more tests for f-strings.

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation in the README file.
- [ ] I have updated the documentation accordingly.
- [ ] I have added an entry in NEWS.rst.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/test_format_strings.py [hunk 1]
```diff
@@ -16,14 +16,6 @@ def test_new_format_string(v):
     check(v.used_names, ['a', 'b', 'c', 'd', 'locals'])
 
 
-@pytest.mark.skipif(
-    sys.version_info < (3, 6),
-    reason="needs f-string support (added in Python 3.6)")
-def test_f_string(v):
-    v.scan("f'{a}, {b:0d} {c:<30} {d:.2%}'")
```
Comment (jendrikseipp) [inline tests/test_format_strings.py]: Thanks for the PR! I don't think we need more tests for f-strings. Instead, you can extend the old f-string to include your new examples: `f'{a}, {b:0d} {c:<30} {d:.2%} {e()} {f:{width}.{precision}} {{x:y for (x, y) in ((1, 2), (3, 4))}}'`

File: vulture/core.py [hunk 2]
```diff
@@ -428,6 +428,9 @@ def visit_ClassDef(self, node):
         else:
             self._define(
                 self.defined_classes, node.name, node, ignore=_ignore_class)
+    
+    def visit_Constant(self, node):
+        self.visit_Str(node)
```
Comment (jendrikseipp) [inline vulture/core.py]: This method already exists below.

**Comment by jendrikseipp** [comment]
Thanks!

File: tests/test_format_strings.py [hunk 3]
```diff
@@ -20,8 +20,12 @@ def test_new_format_string(v):
...
     reason="needs f-string support (added in Python 3.6)")
 def test_f_string(v):
-    v.scan("f'{a}, {b:0d} {c:<30} {d:.2%}'")
-    check(v.used_names, ['a', 'b', 'c', 'd'])
+    v.scan('''\
+f'{a}, {b:0d} {c:<30} {d:.2%} {e()} {f:{width}.{precision}}'
+f'{ {x:y for (x, y) in ((1, 2), (3, 4))} }'
+''')
+    check(v.used_names,
+        ['a', 'b', 'c', 'd', 'e', 'f', 'width', 'precision', 'x', 'y'])
```
Comment (RJ722) [inline tests/test_format_strings.py]: Hey, I know this has already been merged, but I had a very minor nitpick to make - should we arrange them alphabetically? (PR ready at a button click if you would :P)

Comment (jendrikseipp) [reply tests/test_format_strings.py]: No need, thanks.

Comment (RJ722) [reply tests/test_format_strings.py]: Ahh, you already did it. Great!

**Pull request merged**